lady_luck.js

```diff
@@ -551,13 +509,11 @@ function wordCountConsider(msg) {
 
 function wcForce(msg, user_id, bp_total, wc_total) {
 	if (user_id == null || isNaN(user_id)) {
-		msg.reply("You must specify the user whose total you are forcing.");
-		return;
+		return msg.reply("You must specify the user whose total you are forcing.");
 	}
 
 	if (bp_total == null || isNaN(bp_total) ) {
-		msg.reply("Must submit a bonus point total, and it must be a number");
-		return;
+		return msg.reply("Must submit a bonus point total, and it must be a number");
```

Comment:
Unused return value.

Reply:
The return line pre-change was used to prevent further code execution, this just removes the unnecessary extra line for the `return`. There isn't any performance or memory penalty

Reply:
Don't return unused values.